inscriptions/ISic001287.xml

```diff
@@ -194,19 +199,20 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140777">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#AL">
                 <ab>
-                    <lb n="1"/>
-
-
-                    Αὐρήλις Βιτάλης νῦν ἐνθάδε<lb n="2"/>κεῖμαι, | κόσμου πλάνην προ-<lb n="3"/>λιπὼν εἰς αἰώνιον
-                    οἶκον ἀ-<lb n="4"/>νελθών, | μηδένα λυπήσας ὁ-<lb n="5"/>μαλὸν βίον ὧδε διάγω· ἔζησα<lb n="6"/>δὲ
-                    ἔτη ξε.
-                    <lb n="7"/>
+                    <lb n="1"/><persName type="attested"><name type="nomen">Αὐρήλις</name> <name type="cognomen">Βιτάλης</name></persName> <w>νῦν</w> <w>ἐνθάδε</w>
+                    <lb n="2"/><w>κεῖμαι</w> <w>κόσμου</w> <w>πλάνην</w> <w>προ
+                    <lb n="3" break="no"/>λι<hi rend="ligature">πὼν</hi></w> <w>εἰς</w> <w>αἰώνιον</w> <w>οἶκον</w> <w>ἀ
+                    <lb n="4" break="no"/>νελθ<hi rend="ligature">ών</hi></w> <w><hi rend="ligature">μη</hi>δένα</w> <w>λυ<hi rend="ligature">πή</hi>σας</w> <w>ὁ
+                    <lb n="5" break="no"/>μαλὸν</w> <w>βίον</w> <w>ὧδε</w> <w>διάγω</w> <w>ἔζησα</w>
```

Comment:
So, the way that you encode ligatures across word breaks is to split it in two ligatures and give each an xml:id; then at the end of the text or line (exact location doesn't matter) - put a `<link target="#id1 #id2"/>`
E.g. `<w>βίο<hi rend="ligature" xml:id="lig1">ν</hi></w> <w><hi rend="ligature" xml:id="lig2">ὧ</hi>δε</w> <link target="#lig1 #lig2"/>`